python/ray/train/torch/torch_checkpoint.py

```diff
@@ -28,15 +28,22 @@ def from_model(
         *,
         preprocessor: Optional["Preprocessor"] = None,
     ) -> "TorchCheckpoint":
-        """Create a :py:class:`~ray.air.checkpoint.Checkpoint` that stores a Torch
-        model.
+        """Create a :class:`~ray.air.checkpoint.Checkpoint` that stores a Torch model.
+
+        .. warning::
```

Comment:
does this need to be a warning?

Reply:
Eh...maybe note. Changed it to `.. note::`.

Fixed.